xlml/utils/metric.py

```diff
@@ -405,7 +405,9 @@ def get_gke_job_status(benchmark_id: str) -> bigquery.JobStatus:
   return bigquery.JobStatus.FAILED
 
 
-def get_gce_job_status(benchmark_id: str) -> bigquery.JobStatus:
+def get_gce_job_status(
+    task_test_config: test_config.TestConfig[test_config.Tpu], use_startup_script: bool
+) -> bigquery.JobStatus:
```

Comment:
Could you help update the logic in the comment?

Reply:
Done.